index.js

```diff
@@ -887,6 +887,19 @@ var QUERIES = [
       return result
     }
   },
+  {
+    regexp: /^feature\s+(\w+)$/,
+    select: function (context, feature) {
+      env.loadFeature(browserslist.usage, feature)
+      var usage = browserslist.usage[feature]
+      return Object.keys(usage).reduce(function (result, version) {
+        if (usage[version] !== 'n') {
```

Comment:
What if it will be `n #1` (not, with a comment)?

Reply:
It would fail :-) I did just a simple implementation with basic support, correct one would need to actually parse the info fields, I'm working on that.

Reply:
Finally I've just changed the conditional to use `indexOf()` instead of a strict checking for `n`. Where can I find info about the meaning of all the flags that can be set? They are defined at https://github.com/ben-eb/caniuse-lite/blob/master/src/lib/supported.js but don't know the meaning of all of them... :-/ With that info, I could change the regexp so it could be possible to match `supports partially XXX` or `supports not deprecated YYY`, for example.

Reply:
https://github.com/Fyrd/caniuse/blob/master/CONTRIBUTING.md#supported-changes

Found it! :-)

> stats — The collection of support data for a given set of browsers/versions. Only the support value strings can be modified; additional versions cannot be added. Values are space-separated characters with these meanings, and must answer the question "Can I use the feature by default?":
y - (Y)es, supported by default
a - (A)lmost supported (aka Partial support)
n - (N)o support, or disabled by default
p - No support, but has (P)olyfill
u - Support (u)nknown
x - Requires prefi(x) to work
d - (D)isabled by default (need to enable flag or something)
#n - Where n is a number, starting with 1, corresponds to the notes_by_num note. For example: "42":"y #1" means version 42 is supported by default and see note 1.

Reply:
I suggest:

```js
let support = usage[version][0]
if (support !== 'n' && support !== 'p' && support !== 'd')
```

Reply:
I've reconsider this, at first I wanted "support, no matter how", but your proposal gave the idea that's better to filter for "native support provided by the browser itself, without intervention from developer or user". That's lead us only to full and partial support, both with prefix or not, so I've changed the conditional to `flags.indexOf('y') >= 0 || flags.indexOf('a') >= 0` (I've inverted the conditional since this way was smaller, with the new conditions).